packages/react/menu/src/Menu.tsx

```diff
@@ -453,6 +446,11 @@ interface MenuContentImplProps
    * (default: `true`)
    */
   portalled?: boolean;
+  disableOutsidePointerEvents?: DismissableLayerProps['disableOutsidePointerEvents'];
+  onEscapeKeyDown?: DismissableLayerProps['onEscapeKeyDown'];
+  onPointerDownOutside?: DismissableLayerProps['onPointerDownOutside'];
+  onFocusOutside?: DismissableLayerProps['onFocusOutside'];
+  onInteractOutside?: DismissableLayerProps['onInteractOutside'];
```

Comment:
Put these inline instead because the props aren't spread on the `DismissableLayer` part and meant I didn't need to omit `dir` prop from two places.